src/controllers/DappsStakingController.ts

```diff
@@ -0,0 +1,15 @@
+import express, { Request, Response } from 'express';
+import { injectable, inject } from 'inversify';
+import { IDappsStakingService } from '../services/DappsStakingService';
+import { IControllerBase } from './IControllerBase';
+
+@injectable()
+export class DappsStakingController implements IControllerBase {
+    constructor(@inject('DappsStakingService') private _stakingService: IDappsStakingService) {}
+
+    public register(app: express.Application): void {
+        app.route('/api/:network/dapps/apr').get(async (req: Request, res: Response) => {
```

Comment:
`dapps` is a bit abstract. How about `/api/:network/dapps-staking/apr` or `/api/:network/staking/apr`?

Reply:
I put dapps-staking